src/commands/instance/delete.ts

```diff
@@ -0,0 +1,36 @@
+import {flags} from '@oclif/command'
+import cli from 'cli-ux'
+
+import Command from '../../root-command'
+
+export default class InstanceDelete extends Command {
+  static description = 'Delete an instance'
+
+  static flags = {
+    ...Command.flags,
+    'keep-data': flags.boolean({description: 'Do not delete services\' persistent data'}),
```

Comment:
I feel it quite dangerous that the default value is actually removing the data..
Could we do the opposite and keep the data by default?
Renamed this flag by "remove-data"

Reply:
done, just called it `delete-data` to stay consistent with the api